jdaviz/configs/default/plugins/model_fitting/tests/test_fitting.py

```diff
@@ -152,7 +160,7 @@ def test_cube_fitting_backend(unc):
     elif unc is None:
         uncertainties = None
 
-    spectrum = Spectrum1D(flux=flux_cube*u.Jy, spectral_axis=x*u.um,
+    spectrum = Spectrum1D(flux=flux_cube*u.Jy, spectral_axis=x*u.um, wcs=cube_wcs,
```

Comment:
Is it redundant to initialize with a `spectral_axis` and WCS with a spectral axis?

Reply:
You can provide both, but the WCS one should take precedence; not sure if it actually wipes the original `spectral_axis` here
https://github.com/astropy/specutils/blob/6eb7f96498072882c97763d4cd10e07cf81b6d33/specutils/spectra/spectrum1d.py#L268-L270

Reply:
Yes, I think so. Removed. Thanks!